load_tests/load_test.py

```diff
@@ -6,36 +6,34 @@
 import subprocess
 from datetime import datetime, timezone
 
-PLATFORM = os.environ['PLATFORM']
+PLATFORM = os.environ['PLATFORM'].lower()
 OUTPUT_PLUGIN = os.environ['OUTPUT_PLUGIN'].lower()
 TESTING_RESOURCES_STACK_NAME = os.environ['TESTING_RESOURCES_STACK_NAME']
 PREFIX = os.environ['PREFIX']
+EKS_CLUSTER_NAME = os.environ['EKS_CLUSTER_NAME']
 LOGGER_RUN_TIME_IN_SECOND = 600
 BUFFER_TIME_IN_SECOND = 300
+NUM_OF_EKS_NODES = 4
```

Comment:
Can you add a README explaining how the tests work and how tests/pods are distributed between nodes and how someone could run these themselves and etc? Either in this PR or in a separate one.